pkg/minikube/cluster/cluster_test.go

```diff
@@ -147,6 +149,48 @@ func TestStartHostExists(t *testing.T) {
 	}
 }
 
+func TestStartHostErrMachineNotExist(t *testing.T) {
+	RegisterMockDriver(t)
+	api := tests.NewMockAPI(t)
+	// Create an incomplete host with machine does not exist error(i.e. User Interrupt Cancel)
+	api.NotExistError = true
+	ih, err := createHost(api, defaultMachineConfig)
+	if err != nil {
```

Comment:
hoe about return a error type like ErrMachineNotFound and compare to that error type, instead checkig if there its not nil ?

Reply:
Added new error ErrorMachineNotExist.